hera_cal/io.py

```diff
@@ -700,10 +697,11 @@ def write_vis(fname, data, lst_array, freq_array, antpos, time_array=None, flags
 
     telescope_location : type=ndarray, telescope location in xyz in ITRF (earth-centered frame).
 
-    integration_time : type=float or ndarray, integration duration in seconds for data_array. This does not necessarily have
-        to be equal to the diff(time_array): for the case of LST-binning, this is not the duration of the LST-bin
-        but the integration time of the pre-binned data. If None, will use diff(time_array) by default.
-        
+    integration_time : type=float or ndarray, integration duration in seconds for data_array.
+        This does not necessarily have to be equal to the diff(time_array): for the case of
+        LST-binning, this is not the duration of the LST-bin but the integration time of the
+        pre-binned data. Default is median(diff(time_array)) in seconds.
```

Comment:
I would clarify here that the total integration time is always `integration_time * nsamples`.

Reply:
sure thing